src/Dashboard/Data/FunctionInstanceSnapshot.cs

```diff
@@ -17,7 +17,7 @@ public class FunctionInstanceSnapshot
 
         public Guid Id { get; set; }
 
-        public Guid HostInstanceId { get; set; }
```

Comment:
Yeah, we need to ensure this isn't breaking. Likely that means manual testing of all the various scenarios. I'm not sure my self from looking at the code. We'll also need to get a clean run of the Dashboard tests, which again is manual currently because they're not yet running on CI.